src/storage/client/StorageClient.cpp

```diff
@@ -502,6 +567,11 @@ folly::SemiFuture<StorageRpcResponse<storage::cpp2::GeneralResponse>>
 StorageClient::get(GraphSpaceID space,
                    const std::vector<std::string>& keys,
                    folly::EventBase* evb) {
+    if (!loadLeaderBefore_ && !loadLeader()) {
```

Comment:
Could we move loadLeader inside this->leader() function.

Reply:
fixed